### ISSUE TITLE
subdomain request for my personal netlify app

### ISSUE THREAD

**Issue by Larissa-Cury** [rbind/support#920]
## Netlify website address

larissascury.netlify.app

## Preferred rbind.io subdomain

larissascury.rbind.io

### Agreement

- [x] By submitting this request, I promise I will at least write one blog post or create one web page on my website after I get the rbind.io subdomain.

**Comment by nanxstats** [comment]
@Larissa-Cury We have configured the rbind subdomain you requested. Please:

1. Add the rbind subdomain in your Netlify account as the custom domain to your site, as is shown below. Simply use your `subdomain.rbind.io` as the record, no `www` needed.

![](https://user-images.githubusercontent.com/199363/45705707-47b6c580-bb48-11e8-8049-0859c7cc0159.png)

2. There might be messages saying "DNS verification failed" or "check DNS configuration" on Netlify after adding the rbind subdomain -- they can be safely ignored as long as the site renders properly.

3. You may also find the last two sections of [this post](https://yihui.name/en/2017/11/301-redirect/) helpful for redirecting HTTP traffic to HTTPS automatically.

Thanks!
-Nan

**Comment by Larissa-Cury** [comment]
Hi, Nan! Thank you very much. I guess that I did something wrong, now the site is crashing. 

I've added the subdomain to Netlify: 

![image](https://user-images.githubusercontent.com/89667031/199477896-ca7eb30e-78f6-4eb5-97dd-d9daab1bce7f.png)

Then I added it to Hugo Apero confi.toml file:

![image](https://user-images.githubusercontent.com/89667031/199478031-c66db11f-a688-4672-845c-5d4a0289dcf2.png)

but now the site is crashing...Any ideas why? 

Ps: I wasn't sure If I needed to follow step 3 (maybe that's the problem). I was waiting the rbind domain to start actually sharing the website, so do I need to change that too ?

**Comment by Larissa-Cury** [comment]
Hi, Nan! Thank you very much! 😁 I guess that everything is working just fine! (hopefully, I didn't do anything wrong)

I have a question concerning Step 3. I don't know if I should follow it or not? I mean, I'll start to actually share the website now, I was waiting for the rbind subdomain. Before that, I only made one tt post with the Netlify domain and showed the site to a few friends. Do I have to go through this step as well?

If I got it right, the idea of redirecting is for those who already had a former website and want to redirect to the newest domain? Am I right? (this is my very first site, so I'm really sorry if it's a silly question)
